### PR TITLE
Exemples -> Examples

### PR DESCRIPTION
**Pull request by RelativeSure**

### CODE REVIEW

File: exemples/binary-vm/files/traefik/conf/whoami.yml
Comment (maxlerebourg) [inline exemples/binary-vm/files/traefik/conf/whoami.yml:1]: Why remove this file ?

**Comment by mathieuHa** [comment]
Thanks @RelativeSure

I appreciate the rename of exemple (french version) to example.

Like @maxlerebourg mentionned the file `whoami.yml` is still needed and called by Traefik in the example here:
https://github.com/maxlerebourg/crowdsec-bouncer-traefik-plugin/blob/f2aea695fca93fca7f4a845527ac56ba290a9922/exemples/binary-vm/files/traefik/traefik.yml#L24

Otherwise seems good to me.

File: exemples/binary-vm/files/traefik/conf/whoami.yml
Comment (RelativeSure) [reply exemples/binary-vm/files/traefik/conf/whoami.yml:1]: Good question, idk why it removed that. Lemme put it back

Comment (RelativeSure) [reply exemples/binary-vm/files/traefik/conf/whoami.yml:1]: .gitignore file is ignoring conf folder

**Comment by RelativeSure** [comment]
> Thanks @RelativeSure
> 
> I appreciate the rename of exemple (french version) to example.
> 
> Like @maxlerebourg mentionned the file `whoami.yml` is still needed and called by Traefik in the example here:
> 
> https://github.com/maxlerebourg/crowdsec-bouncer-traefik-plugin/blob/f2aea695fca93fca7f4a845527ac56ba290a9922/exemples/binary-vm/files/traefik/traefik.yml#L24
> 
> Otherwise seems good to me.

.gitignore is ignoring conf folder, thats why it's removed. I have put it back now.

File: vendor/github.com/maxlerebourg/simpleredis/README.md [hunk 1]
```diff
@@ -28,4 +28,4 @@ if err != nil {
 
 ## Author
 Max Lerebourg @ [Primadviz.com](https://primadviz.com)
-Mathieu Hanotaux
\ No newline at end of file
+Mathieu Hanotaux
```
Comment (maxlerebourg) [inline vendor/github.com/maxlerebourg/simpleredis/README.md:31]: Your linter modified file inside vendor folder. Not good practice

**Comment by maxlerebourg** [comment]
Merged, thank you very much for your contribution, it's really appreciated.

File: vendor/github.com/maxlerebourg/simpleredis/README.md (hunk 1 above)
Comment (RelativeSure) [reply vendor/github.com/maxlerebourg/simpleredis/README.md:31]: It's a final newline from my vscode setup. I am not using a linter.

**Pull request merged**